include/OverlayRendering.h

```diff
@@ -0,0 +1,63 @@
+#pragma once
+
+#include "Access.h"
+#include "Definitions.h"
+#include "Geometry.h"
+
+struct OverlayLine
+{
+	geometry::Vec2 start;
+	geometry::Vec2 end;
+	uint32_t       colour;
+
+	OverlayLine(geometry::Vec2 start, geometry::Vec2 end, uint32_t colour)
+		: start(start)
+		, end(end)
+		, colour(colour)
+	{ }
+};
+
+struct OverlayText
+{
+	std::string    text;
+	geometry::Vec2 position;
+};
+
+struct OverlayTest
+{
+	uint32_t magic = 0; /* Define test number. */
+
+	OverlayTest(uint32_t magic)
+		: magic(magic)
+	{ }
+};
+
+namespace core
+{
+	struct RenderSignal;
+	struct Render2Signal;
```

Comment:
Rename to `PostRenderSignal`

Reply:
> Rename to `PostRenderSignal`

`PostRenderSignal` already exists, would `PostProcessRenderSignal` be a good fit? This new signal has parameters, I don't think it would be a good idea to simply rename stuff here.

Reply:
PostProcessRenderSignal is a good name, go for it!